miner/miner_test.go

```diff
@@ -75,11 +67,11 @@ func TestCloudBase(t *testing.T) {
 
 	CloudBase(b, bc, w.Public())
```

Comment:
hahaha sure I thought I was being clever but it is sort of hard to understand isn't it 😂